apps/web/ui/links/link-details-column.tsx

```diff
@@ -148,29 +148,22 @@ function AnalyticsBadge({ link }: { link: ResponseLink }) {
         id: "clicks",
         icon: CursorRays,
         value: link.clicks,
-        label: conversionEnabled
-          ? undefined
-          : (value) => pluralize("click", value),
         iconClassName: "data-[active=true]:text-blue-500",
       },
-      ...(conversionEnabled
-        ? [
-            {
-              id: "leads",
-              icon: UserCheck,
-              value: link.leads,
-              className: "hidden sm:flex",
-              iconClassName: "data-[active=true]:text-purple-500",
-            },
-            {
-              id: "sales",
-              icon: InvoiceDollar,
-              value: link.saleAmount,
-              className: "hidden sm:flex",
-              iconClassName: "data-[active=true]:text-teal-500",
-            },
-          ]
-        : []),
```

Comment:
Is it intentional that we're now showing leads & sales stats on every link card, even for free/pro users, or should we limit that to only links with `trackConversion=true`?

Reply:
nope yeah it's intentional (to drive more awareness)! we can revert tho if user complain